app/models/event.rb

```diff
@@ -21,7 +21,7 @@ def ical
     e
   end
 
-  def as_json
+  def as_json(options = {})
```

Comment:
Hmm, perhaps this is the issue? It should just accept attributes to be able to run it.